docs/unified-test-documentation/dasharo-security/200-tpm-support.md

```diff
@@ -404,3 +404,188 @@ Example output:
 ```text
 SpecVersion=2.0
 ```
+
+## TPM013.001 Signing the file (Ubuntu 22.04)
+
+**Test description**
+
+This test aims to verify that the TPM supports file signing.
+
+**Test configuration data**
+
+1. `FIRMWARE` = Dasharo
+1. `OPERATING_SYSTEM` = Ubuntu 22.04
+
+**Test setup**
+
+1. Proceed with the
+    [Test cases common documentation](#test-cases-common-documentation) section.
+1. Install the `tpm2-tools` package: `sudo apt install tpm2-tools`.
+
+**Test steps**
+
+1. Power on the DUT.
+1. Boot into the system.
+1. Log into the system by using the proper login and password.
+1. Open the terminal and run the following set of commands to generate the
+   required keys and files:
+
+    ```bash
+    sudo tpm2_clear
```

Comment:
@Pokisiekk is this command really used for generating keys? If we need to clear the TPM before starting the procedure, we should add this to the test setup (or at least to a different point).

Reply:
I move tpm2_clear to a different point because I tested in this way.
It is possible that at some stage of booting or logging into the system, something will be saved that will affect the tests, hence tpm2_clear should be executing just before specific commands.

Reply:
okay, thanks for the answer